pkg/bloomgateway/querier.go

```diff
@@ -57,8 +58,12 @@ type BloomQuerier struct {
 	metrics *querierMetrics
 }
 
-func NewQuerier(c Client, logger log.Logger) *BloomQuerier {
-	return &BloomQuerier{c: c, logger: logger}
+func NewQuerier(c Client, r prometheus.Registerer, logger log.Logger) *BloomQuerier {
+	return &BloomQuerier{
+		c:       c,
+		metrics: newQuerierMetrics(r, constants.Loki, "bloom_gateway_querier"),
```

Comment:
Add string as constant?

Reply:
Can do, but it's really only used here.

Reply:
https://github.com/grafana/loki/pull/12036/commits/99277d2c49bb7fff9b4600318802820690597dd5